inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -257,7 +257,7 @@ public function display_no_table_notice() {
 			$main_message,
 			'<strong>WP Rocket</strong>',
 			$this->used_css->get_name(),
-			'<a href="' . $this->get_support_url() . '" target="_blank" rel="noopener">',
+			'<a href="' . esc_url( __( 'https://docs.wp-rocket.me/article/1828-could-not-create-the-rucss-usedcss-table/?utm_source=wp_plugin&utm_medium=wp_rocket', 'rocket' ) ) . '" target="_blank" rel="noopener">',
```

Comment:
It would be better to use the Beacon object to get the link, it will automatically adapt for french or english version that way.

If the link is not yet part of the Beacon object, you can add it, and ask Piotr for the URL in FR.

Reply:
FR url is not available at the moment